src/LegendDSP.jl

```diff
@@ -49,6 +49,7 @@ include("ml.jl")
 include("dsp_ml_routines.jl")
 include("multi_intersect.jl")
 include("moving_window_multi.jl")
+include("derivative.jl")
```

Comment:
I need this filter in all `dsp_routines` so could you move this up and include all fitlers first?

Reply:
Done!